.travis.yml

```diff
@@ -89,15 +69,26 @@ matrix:
         - BUILD_TYPE=Release
         - PREFIX_PATH="/usr/local/opt/qt/lib/cmake"
         - BUILD_DOC=Off
+        - MATRIX_EVAL=""
       os: osx
       compiler: clang
 
+#    # mac gcc Release
+#    - env:
+#        - BUILD_TYPE=Release
+#        - PREFIX_PATH="/usr/local/opt/qt/lib/cmake"
+#        - BUILD_DOC=Off
+#	# see https://github.com/Homebrew/brew/issues/1742
+#        - MATRIX_EVAL="rm /usr/local/include/c++ && brew install gcc && CC=gcc-8 && CXX=g++-8"
+#      os: osx
```

Comment:
Why is this commented ? if not working, might just remove theses lines ?

Reply:
Fixed by 8eb878c